src/renderers/webgpu/utils/WebGPUTextureUtils.js

```diff
@@ -266,6 +266,7 @@ class WebGPUTextureUtils {
 
 			msaaTextureDescriptorGPU.label = msaaTextureDescriptorGPU.label + '-msaa';
 			msaaTextureDescriptorGPU.sampleCount = samples;
+			msaaTextureDescriptorGPU.mipLevelCount = 1; // See https://developer.mozilla.org/en-US/docs/Web/API/GPUDevice/createTexture
```

Comment:
Can we link directly to the spec instead? https://www.w3.org/TR/webgpu/#texture-creation